data/mods/blazemod/blaze_ammo_recipes.json

```diff
@@ -97,8 +97,8 @@
   ],
   "components": [
     [
-      [ "can_food_unsealed", 1 ],
-      [ "can_drink_unsealed", 1 ],
+      [ "can_food", 1 ],
+      [ "can_drink", 1 ],
```

Comment:
This should be "can_food_unsealed" and same for "can_drink_unsealed". Otherwise it won't ever be craftable, since you'd need an empty but sealed can.